### PR TITLE
Software Lights on!

### PR DESCRIPTION
**Pull request by AMurkin**
New software: "Lights on!"
image

image

See [Lights Out (game)](https://en.wikipedia.org/wiki/Lights_Out_(game))

### CODE REVIEW

File: src/iuse_software_lightson.cpp [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    draw_border( w_border );
+
+    input_context ctxt( "LIGHTSON" );
+    ctxt.register_directions();
+    ctxt.register_action( "TOGGLE" );
+    ctxt.register_action( "RESET" );
+    ctxt.register_action( "QUIT" );
+
+    std::vector<std::string> shortcuts;
+    shortcuts.push_back( _( "<5>toggle lights" ) ); // '5': toggle
```
Comment (Night-Pryanik) [inline src/iuse_software_lightson.cpp]: Maybe add space after `<5>`? Also, capitalization? Also, maybe change shortcut to spacebar?

**Comment by ZhilkinSerg** [comment]
I suggest adding another key to randomize level without restart of the game.

**Comment by AMurkin** [comment]
I want interface to be as simple as possible.

**Comment by ZhilkinSerg** [comment]
> I want interface to be as simple as possible.

I see, but interface would be hardly changed - just another keybinding and label in the header.

image

Anyway it is just a suggestion and does not prevent PR from merging.

***

image

image

**Pull request merged**